### DISCUSSION TITLE
Incorrect CC name + Suggestion

### DISCUSSION THREAD

**Discussion by dannybullo** [djipco/webmidi#324] [General]
Hi,

I realized you have an incorrect CC name:

**registeredparametercoarse: 100
registeredparameterfine: 101**

This is _reversed_ as the correct MIDI CC Names are:

**100 Registered Parameter (LSB)
101 Registered Parameter (MSB)**

Suggestion:

Can you change the Array MIDI_CONTROL_CHANGE_MESSAGES to have more friendly names so that getCcNameByNumber() returns more "readable" values?

for example:
use **"Registered Parameter (LSB)"** instead of **"registeredparameterfine"**
use **"Sostenuto Pedal "** instead of **"sustenutopedal"**
use **"Bank Select (MSB)"** instead of **"bankselectcoarse"**

Maybe there is an official standardized cc name list. See this.

https://professionalcomposers.com/midi-cc-list/

Thanks!

Danny Bullo

**Comment by djipco** [comment]
You are right. Thanks for reporting this problem @dannybullo 

I created an issue from your message so I can properly track it. We can continue discussing this under issue #325.
